src/registry.cpp

```diff
@@ -36,6 +36,7 @@
 #endif
 
 std::string Registry::ReadStrValue(HKEY hkey, std::string const& key, std::string const& val, REGVIEW view) {
+#ifndef _ARM_
```

Comment:
Do you intend to add registry reading on WINRT in the future?
If not, instead of patching the functions here, the filefinder could be changed and this whole file excluded on ARM.